crates/wiggle/generate/src/config.rs

```diff
@@ -201,6 +202,29 @@ impl Parse for Paths {
         let content;
         let _ = bracketed!(content in input);
         let path_lits: Punctuated<LitStr, Token![,]> = content.parse_terminated(Parse::parse)?;
+
+        /* TODO interpolate env variables here!!!
+                        fn main() {
+            let p = "foo/$BAR/baz";
+            let buf = PathBuf::from(p);
+            let components = buf
+                .iter()
+                .map(|osstr| interpolate(osstr.to_str().expect("always a str")))
+                .collect::<Vec<String>>();
+
+            println!("{:?}", components);
+        }
+
+        fn interpolate(v: &str) -> String {
```

Comment:
Perhaps the [shellexpand crate](https://github.com/netvl/shellexpand)?

Reply:
Great idea, idk why I thought to reinvent this wheel, that looks like a sensible dep.